Algorithms/Mathematical/catalan_numbers.cpp

```diff
@@ -31,13 +33,23 @@ ll n;
 cin>>n;
```

Comment:
no single variables

Reply:
I did not understand what you are trying to say.

Reply:
No single letter variables